app/assets/js/graph.js

```diff
@@ -1,79 +1,70 @@
-import '../../../govuk/vendor/polyfills/Function/prototype/bind'
-import '../../../govuk/vendor/polyfills/Event' // addEventListener and event.target normaliziation
-import { nodeListForEach } from '../../../govuk/common'
 
-function Graph($module) {
-    this.$module = $module
-}
+var $module = document.querySelectorAll('[data-module="idsk-graph"]')[0];
 
-Graph.prototype.init = function () {
-    // Check for module
-    var $module = this.$module
-    if (!$module) {
-        return
+var $radioOptions = $module.querySelectorAll('.idsk-graph__radio');
+if ($radioOptions) {
+    for (let index = 0; index < $radioOptions.length; index++) {
```

Comment:
mali sme uz tak skor dohodu ze vsetky premenne by mali byt typu var, cize 
for (var index = 0; index < $radioOptions.length; index++) {